javaSE/src/main/java/com/smartdevicelink/managers/lifecycle/LifecycleManager.java

```diff
@@ -1145,6 +1145,11 @@ public void getCapability(SystemCapabilityType systemCapabilityType, OnSystemCap
 
         }
 
+        @Override
+        public Object getCapability(SystemCapabilityType systemCapabilityType, OnSystemCapabilityListener scListener, boolean forceUpdate) {
+            return LifecycleManager.this.systemCapabilityManager.getCapability(systemCapabilityType, scListener, forceUpdate);
```

Comment:
There should probably be some null checks here and in the `isCapabilitySupported` methods to make sure that `LifecycleManager.this.systemCapabilityManager` isn't null before simply accessing it.